automation/carbonite-position-check.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
```

Comment:
a little usage doc?

Reply:
Turns out parameterization is not necessary for this